src/agent/configuration_parser/include/configuration_parser.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <toml.hpp>
+
+#include <iostream>
+
+namespace configuration
+{
+    class ConfigurationParser
```

Comment:
I think this should be instantiated in the Agent class.

Reply:
Done